docs/index.rst

```diff
@@ -6,15 +6,18 @@
 
    installing
    configuring
-   changelog
    contributing
-   translations
 
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   changelog
```

Comment:
Is there a reason we're hiding this?

Reply:
For the front page of the docs, the TOC with all the versions just takes up space

Reply:
It would be nice if these type of changes were handled has a separate PR.

Reply:
Generally I agree, but I think this is minor and related.